scipy/linalg/basic.py

```diff
@@ -70,11 +70,14 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Input data for the right hand side.
     sym_pos : bool, optional
         Assume `a` is symmetric and positive definite. This key is deprecated
-        and assume_a = 'pos' keyword is recommended instead. The functionality
-        is the same. It will be removed in the future.
+        and ``assume_a='pos'`` is recommended instead. The functionality is
+        the same. It will be removed in the future.
+        Default is False.
     lower : bool, optional
-        If True, only the data contained in the lower triangle of `a`. Default
-        is to use upper triangle. (ignored for ``'gen'``)
+        If True, the calulation is based only on the data in the lower triangle of
+        matrix `a` instead of data in the upper triangle.
+        Ignored if ``assume_a == 'gen'``.
+        Default is False.
```

Comment:
It is implicit, but consider writing out what this means: the calculation uses only data in the upper triangle; entries below the diagonal are ignored.